<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border:none;
	background-color: transparent;
	background: none;
	padding:0;
	margin:0;
	color:#fff;
}
#centralwidget{
	background-color: rgb(31, 35, 42);
}
#leftMenu{
	background-color: #16191d;
}
#leftMenu QPushButton{
	text-align:left;
	padding:5px 10px;
	border-top-left-radius:10px;
	border-top-bottom-radius:10px;
}
#header{
	background-color: #2c313c;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignLeft">
     <widget class="QWidget" name="left" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="leftMenu" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="menuBtn">
               <property name="toolTip">
                <string>menu</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="icon.qrc">
                 <normaloff>:/new/prefix1/feather/align-justify.svg</normaloff>:/new/prefix1/feather/align-justify.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignBottom">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QPushButton" name="examBtn">
               <property name="toolTip">
                <string>exam</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Egzamin</string>
               </property>
               <property name="icon">
                <iconset resource="icon.qrc">
                 <normaloff>:/new/prefix1/feather/slack.svg</normaloff>:/new/prefix1/feather/slack.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="studentBtn">
               <property name="toolTip">
                <string>student</string>
               </property>
               <property name="text">
                <string>Student</string>
               </property>
               <property name="icon">
                <iconset resource="icon.qrc">
                 <normaloff>:/new/prefix1/feather/user.svg</normaloff>:/new/prefix1/feather/user.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="questionBtn">
               <property name="toolTip">
                <string>question</string>
               </property>
               <property name="text">
                <string>Pliki</string>
               </property>
               <property name="icon">
                <iconset resource="icon.qrc">
                 <normaloff>:/new/prefix1/feather/file.svg</normaloff>:/new/prefix1/feather/file.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="right" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QWidget" name="header" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>EGZAMIN</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="toolTip">
                <string>minimalize</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="icon.qrc">
                 <normaloff>:/new/prefix1/feather/minus.svg</normaloff>:/new/prefix1/feather/minus.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="toolTip">
                <string>restore</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="icon.qrc">
                 <normaloff>:/new/prefix1/feather/square.svg</normaloff>:/new/prefix1/feather/square.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="toolTip">
                <string>close</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="icon.qrc">
                 <normaloff>:/new/prefix1/feather/x.svg</normaloff>:/new/prefix1/feather/x.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="mainBody" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="styleSheet">
             <string notr="true">	background-color: rgb(31, 35, 42);</string>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="page">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <pointsize>30</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>HOME</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QWidget" name="widget" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item alignment="Qt::AlignLeft">
                  <widget class="QListWidget" name="studCheckList"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="updateList">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">	background-color: #2c313c;</string>
                   </property>
                   <property name="text">
                    <string>Aktualizuj Liste</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="chooseStud">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">	background-color: #2c313c;</string>
                   </property>
                   <property name="text">
                    <string>Wybierz Studenta</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_2" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QWidget" name="widget_3" native="true"/>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_4" native="true">
                   <widget class="QPushButton" name="runExam">
                    <property name="geometry">
                     <rect>
                      <x>40</x>
                      <y>150</y>
                      <width>201</width>
                      <height>24</height>
                     </rect>
                    </property>
                    <property name="text">
                     <string>Rozpcznij Egzamin</string>
                    </property>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_3">
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QWidget" name="left_2" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QTextEdit" name="studList"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="wczStudBtn">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">	background-color: #2c313c;</string>
                   </property>
                   <property name="text">
                    <string>Wczytaj Studentów</string>
                   </property>
                   <property name="icon">
                    <iconset resource="icon.qrc">
                     <normaloff>:/new/prefix1/feather/user-plus.svg</normaloff>:/new/prefix1/feather/user-plus.svg</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="right_2" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QTextEdit" name="questionList"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="wczPytBtn">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">	background-color: #2c313c;</string>
                   </property>
                   <property name="text">
                    <string>Wczytaj Pytania</string>
                   </property>
                   <property name="icon">
                    <iconset resource="icon.qrc">
                     <normaloff>:/new/prefix1/feather/upload-cloud.svg</normaloff>:/new/prefix1/feather/upload-cloud.svg</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
